internal/store/store.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/mongodb-forks/digest"
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
 	"github.com/mongodb/mongodb-atlas-cli/internal/log"
+	apiv2 "go.mongodb.org/atlas/api/v1alpha"
```

Comment:
do we intentionally load `v1alpha` as `apiv2`?

Reply:
Good catch. It is confusing.
v1alpha is current client version. 
v2 is an atlas api version it targets.

I think we will need to settle with different names to minimize search and replace in future. Something like `atlasapi` or `atlasv2`

Reply:
Indeed it isn't very clear. How about prefixing all apis with `api` and all clients with `client`?

e.g. `clientv1alpa`, `apiv2` etc

Reply:
it is even more confusing as golang recomends to have `api` prefix for parts that are public interface.

To avoid confusion I settled with atlasv2 as name of the API and it's version. This will also allow us to minimize amount of changes (not including actual client version we can change that in the cli by changing only imports)